source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -162,20 +162,31 @@ public static void ApplyBleed(CollectionTypeElement collectionElement, LinearLay
         {
             int padding = Util.dpToPixels(context, hostConfig.GetSpacing().getPaddingSpacing());
             LinearLayout.LayoutParams layoutParams = (LinearLayout.LayoutParams) collectionElementView.getLayoutParams();
-            int marginLeft = layoutParams.leftMargin, marginRight = layoutParams.rightMargin;
+            int marginLeft = layoutParams.leftMargin, marginRight = layoutParams.rightMargin, marginTop = layoutParams.topMargin, marginBottom = layoutParams.bottomMargin;
 
             ContainerBleedDirection bleedDirection = collectionElement.GetBleedDirection();
-            if (bleedDirection == ContainerBleedDirection.BleedToLeading || bleedDirection == ContainerBleedDirection.BleedToBothEdges)
+
+            if ((bleedDirection.swigValue() & ContainerBleedDirection.BleedLeft.swigValue()) != ContainerBleedDirection.BleedRestricted.swigValue())
```

Comment:
>!= ContainerBleedDirection.BleedRestricted.swigValue() [](start = 93, length = 54)

What is this for?

Reply:
That's to check that the value is not zero

---
In reply to: [282597046](https://github.com/microsoft/AdaptiveCards/pull/2867#discussion_r282597046) [](ancestors = 282597046)